lib/berlim/accounts/taxi.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Berlim.Accounts.Taxi do
+  @moduledoc false
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Berlim.Sales.{Order, Customer}
+
+  schema "taxis" do
+    field(:cpf, :string)
+    field(:email, :string)
```

Comment:
o campo `email` aparece em quase todos os modules, tá certo isso? `Customer` vai ter um email, e o `Taxi` dele tbm?

Reply:
`Taxi.email` vai receber as notificações referentes a empresa (recibo da central por ex) `Customer.email` faz parte dos dados que vão ser enviados para a api de cartão de credito. Eu pensei em deixar o email somente no customer, mas quem paga (`Customer`) não necessariamente é a mesma pessoa do `Taxi`. @thiagogsr precisamos enviar email para o moip? se não vou tirar e deixar somente no `Taxi`

Reply:
veja a doc do moip, não sei de cabeça

o que eu removeria é a relação de taxi com customer, customer só pertence a order, não tem ligação nenhuma com o taxi

Reply:
hm, toda vez que for fazer um novo pagamento e gerar uma nova order, tem que criar um novo customer? @thiagogsr

Reply:
negativo, só recuperar o customer da última order paga, e permitir que ele altere, pois ele pode pagar em outro cartão de outra pessoa

Reply:
makes sense, ajustarei!